debian/update.sh

```diff
@@ -67,7 +67,8 @@ INSTDIR=$(dirname "$0")
 if [ "${INSTDIR#/}" == "$INSTDIR" ] ; then INSTDIR="$PWD/$INSTDIR" ; fi
 INSTDIR=${INSTDIR%%/debian}
 BUILDDIR_TEMPLATE=$INSTDIR/kbuild
-export KBUILD_BUILD_TIMESTAMP=$(date --rfc-2822)
+KBUILD_BUILD_TIMESTAMP="$(dpkg-parsechangelog -STimestamp)"
```

Comment:
```suggestion
KBUILD_BUILD_TIMESTAMP=${KBUILD_BUILD_TIMESTAMP:-"$(dpkg-parsechangelog -STimestamp)"}
```
This would allow the CI to override the build date for snapshot builds.

Reply:
Don't we modify the changelog file?

Reply:
You're right. nvm :-)